src/@next/utilities/colors/red.ts

```diff
@@ -1,6 +1,6 @@
 export const Brand = '#EC272B';
-export const B100 = '#FFC7C2';
-export const B99 = '#FC6568';
+export const B100 = '#FFEEEE';
```

Comment:
@michac789 I just realized that the red-B100 appears too subtle in the badge component. Would it be possible to adjust red-B100 from #FFEEEE to #FFE5E6?

Reply:
sure!